src/Controllers/HelloWorldController.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Text.Encodings.Web;
+
+namespace TrainingNet.Controllers
+{
+    [Route("[controller]")]
+    public class HelloWorldController : Controller
+    {
+        // 
+        // GET: /HelloWorld/
+
```

Comment:
It's okay to have an empty line separating the methods, but the comments should be right above the method
```
public class HelloWorldController : Controller
{
    // GET: /HelloWorld/
    [HttpGet("")]
    public IActionResult Index()
    {
        return View();
    }
    
    // GET: /HelloWorld/Welcome/ 
    [HttpGet("Welcome/{id?}")]
---
}
```